omegaconf/resolvers/oc/__init__.py

```diff
@@ -92,10 +94,25 @@ def deprecated(
     return target_node
 
 
+def select(
+    key: str,
+    default: Any = _DEFAULT_SELECT_MARKER_,
+    *,
+    _parent_: Container,
+) -> Any:
+    from omegaconf._impl import select_value
+
+    if default is _DEFAULT_SELECT_MARKER_:
+        return select_value(cfg=_parent_, key=key, absolute_key=True)
+    else:
+        return select_value(cfg=_parent_, key=key, absolute_key=True, default=default)
```

Comment:
Why use a different marker?
```suggestion
def select(
    key: str,
    default: Any = _DEFAULT_MARKER_,
    *,
    _parent_: Container,
) -> Any:
    from omegaconf._impl import select_value

    return select_value(cfg=_parent_, key=key, absolute_key=True, default=default)
```

Reply:
good point. Initially I was passing it into select_value.

Reply:
I think it can be simplified further (see my suggestion: no more `if`)

Reply:
yeah, for some reason I thought using the same marker object select_value is using would cause issues.